webtester-support-hamcrest/src/main/java/info/novatec/testit/webtester/support/hamcrest/GenericTextFieldMatcher.java

```diff
@@ -0,0 +1,48 @@
+package info.novatec.testit.webtester.support.hamcrest;
+
+import info.novatec.testit.webtester.pageobjects.GenericTextField;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+
+/**
+ * Contains {@link Matcher matcher} for {@link GenericTextField generic text fields}.
+ *
+ * @since 1.2
+ */
+public class GenericTextFieldMatcher extends PageObjectMatcher {
```

Comment:
And one more thing: please turn the constructor private (instead of protected), so that codacy is happy?